test/config/mosdepth-coverage.config

```diff
@@ -0,0 +1,51 @@
+// EXECUTION SETTINGS AND GLOBAL DEFAULTS
+
+// External config files import. DO NOT MODIFY THESE LINES!
+includeConfig "${projectDir}/config/default.config"
+includeConfig "${projectDir}/config/methods.config"
+includeConfig "${projectDir}/nextflow.config"
+
+
+// Inputs/parameters of the pipeline
+params {
+    algorithm = ['mosdepth_coverage'] // 'fastqc', 'samtools_stats', 'mosdepth_coverage', 'collectwgsmetrics', 'qualimap_bamqc'
+    reference = '/hot/resource/reference-genome/GRCh38-BI-20160721/Homo_sapiens_assembly38.fasta'
+    blcds_registered_dataset = false // if you want the output to be registered
+    save_intermediate_files = true
+
+    // SAMtools stats options
+    stats_remove_duplicates = false
+    stats_additional_options = ''
+
+    // mosdepth window-base coverage options
+    mosdepth_use_fast_algorithm = false
+    mosdepth_window_size = 500
+    mosdepth_per_base_output = true
+    mosdepth_additional_options = ''
+
+    // Picard CollectWgsMetrics options
+    cwm_coverage_cap = 1000
+    cwm_minimum_mapping_quality = 20
+    cwm_minimum_base_quality = 20
+    cwm_additional_options = ''
+    cwm_use_fast_algorithm = false
+
+    // Qualimap bamqc options
+    bamqc_output_format = 'pdf'
+    bamqc_additional_options = ''
+
+    // FastQC options
+    fastqc_additional_options = ''
+
+    // Base resource allocation updater
+      // See README for adding parameters to update the base resource allocations
+}
+
+// Setup the pipeline config. DO NOT REMOVE THIS LINE!
+methods.setup()
+
+process {
+    withName: run_validate_PipeVal {
+        when = false
+    }
+}
```

Comment:
**suggestion**: This can be removed, the test file should be small enough that the validation shouldn't pose a problem